arm-software/embedded/CMakeLists.txt

```diff
@@ -354,6 +360,13 @@ if(LLVM_TOOLCHAIN_C_LIBRARY STREQUAL llvmlibc)
         COMPONENT llvm-toolchain-llvmlibc-configs
     )
 
+    install(
+        FILES
+        ${CMAKE_CURRENT_SOURCE_DIR}/ATfE-SBOM-llvmlibc-overlay.spdx.json
+        DESTINATION .
+        COMPONENT llvm-toolchain-llvmlibc-configs
```

Comment:
Should we use `llvm-toolchain-third-party-licenses` component - config one only seems to contain the cfg file.

Similar for other packages.